### ISSUE TITLE
Can't install Tensorflow.Keras : unable to resolve 'PureHDF'

### ISSUE THREAD

**Issue by IgnacioRodriguez98** [SciSharp/TensorFlow.NET#1096]
### Description

I´m trying to run an LSTM that I made in keras.net to tensroflow.net (in order to look if I can set a random seed in tensorlfow.net) but when I run it I get this error.

**_System.Reflection.ReflectionTypeLoadException: 'Cannot load one or more required types. Retrieve the LoaderExceptions property for more information.'_**

In order to that menssage I decided to uninstall keras.net and install tensorflow.keras. But I have this error when is trying to download and install it.

_**Unable to resolve dependency 'PureHDF'. Source(s) used: 'nuget.org', 'Microsoft Visual Studio Offline Packages'.**_

Somebody has resolve that issue?

**Comment by Oceania2018** [comment]
The message “Microsoft Visual Studio Offline Packages” seems like your network is disconnected.

**Comment by IgnacioRodriguez98** [comment]
> The message “Microsoft Visual Studio Offline Packages” seems like your network is disconnected.

In base to your comment, I try with the internet of my house and with my cellphone and is the same, any other suggestion? Thanks in advice

**Comment by IgnacioRodriguez98** [comment]
> The message “Microsoft Visual Studio Offline Packages” seems like your network is disconnected.

Just checked, the message refers to the only two sources I have, the first is nuget.org and the second is Microsoft Visual Studio Offline Packages.

That makes me think that neither can find the PureHDF dependency, whatever that means
